opm/parser/eclipse/EclipseState/Schedule/Group/GuideRate.hpp

```diff
@@ -75,11 +79,15 @@ struct GuideRateValue {
         return !(*this == other);
     }
 
-    double sim_time;
-    double value;
-    GuideRateModel::Target target;
+    double sim_time { std::numeric_limits<double>::lowest() };
```

Comment:
What is the difference between `..limits<double>::lowest()` and `..limits<double>::min()`

Reply:
For floating-point types, `lowest()` denotes the smallest (most negative) finite value that's representable in the type.  On the other hand, `min()` is smallest (minimum magnitude) **positive** normal number that's representable.  In implementations that use IEEE 754 `lowest() == -max()`, but this relation does not necessarily hold for other floating-point systems.

This is all **very** different from integer types for which `min()` is the most negative value representable.